api/provider/aws/deploy_create.go

```diff
@@ -0,0 +1,131 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (d *DeployProvider) Create(req models.CreateDeployRequest) (*models.Deploy, error) {
+	if req.DeployName == "" {
```

Comment:
Validation should be done on the model, see [CreateEnvironmentRequest](https://github.com/quintilesims/layer0/blob/api-refactor/common/models/create_environment_request.go#L18)

Reply:
✅